cmd/app-mesh-inject/main.go

```diff
@@ -70,6 +70,7 @@ func init() {
 	flag.BoolVar(&cfg.EnableStatsD, "enable-statsd", false, "If enabled, Envoy will send DogStatsD metrics to 127.0.0.1:8125")
 	flag.BoolVar(&cfg.InjectStatsDExporterSidecar, "inject-statsd-exporter-sidecar", false, "This flag is deprecated and does nothing")
 	flag.BoolVar(&cfg.InjectDefault, "inject-default", true, "If enabled, sidecars will be injected in the absence of the corresponding pod annotation")
+	flag.BoolVar(&cfg.InjectFSGroup, "inject-fs-group", true, "If enabled, an fsgroup: 1337 will be injected in the absence of it in pod securityContext")
```

Comment:
I would make the fs group configurable instead of hardcoding 1337, by the way shouldn't it be 65534? Please see https://github.com/aws/aws-app-mesh-inject/issues/116#issuecomment-580135987

Reply:
@stefanprodan thanks, it can be any value to make iam-for-pods work. (fsGroup is an a supplementary group that the user and volume belongs too).
I don't want to make it configurable mainly because it's a workaround to bypass current limitation of iam-for-pods. Users shouldn't relies on the this functionality to set fsGroup for them.